lib/github/ldap.rb

```diff
@@ -10,6 +10,8 @@
 require 'github/ldap/instrumentation'
 require 'github/ldap/member_search'
 require 'github/ldap/membership_validators'
+require 'github/ldap/user_search/default.rb'
+require 'github/ldap/user_search/active_directory.rb'
```

Comment:
We don't need `.rb` here, and would prefer to be consistent with the preceding lines.